juriscraper/opinions/united_states/state/coloctapp.py

```diff
@@ -50,3 +56,31 @@ def _process_html(self) -> None:
                     "status": "Published",
                 }
             )
+
+    def extract_from_text(self, scraped_text: str) -> Dict[str, Any]:
+        """Pass scraped text into function and return data as a dictionary
+        Notes for 'Citation':
+            - Reporter key for this court: 'COA'
+            - Type for a state: 2
```

Comment:
This is great to note, but I think 2 needs more context.  

2 is the value we assign citations that fall into `A citation in a state-based reporter (e.g. Alabama Reports)` in Courtlistener.

Reply:
Updated  note with explanation of the Type value for CL.